types/is-date-object/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for is-date-object 1.0
+// Project: https://github.com/ljharb/is-date-object
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
+
+type InvalidTypes = undefined | null | boolean | string | number | [] | {};
+
+declare function isDateObject(value: Date): true;
+declare function isDateObject(value: InvalidTypes | unknown): false;
```

Comment:
since the first one will match Dates, does that mean these will only match non-Dates?

I kind of think that unknown should always be Boolean, not false - since it’s unknown.

Reply:
Yeah anything other than a date should return a boolean - updated, I believe I've captured all the possible types other than Date in the `InvalidTypes` type